threading/client/client.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/python3
+import socket
+import sys
+import os
+import argparse
+import signal
+import struct
+
+'''
+GLOBAL VARIABLE: FILE_LIST
+    TYPE: list()
+    BRIEF: stores list of files that exists on file server
+    USAGE: used within the RECV_FILE_LIST and GET_FILE_LIST methods
+'''
+file_list = []
+
+def socket_info():
+    '''
+    SOCKET_INFO:
+        PARAMETER: None
+        BRIEF: parses cmdline arguments to get ip address and port info
+        RETURN: server ip address and socket port
+    '''
+    parser = argparse.ArgumentParser()
+    parser.add_argument("host")
+    parser.add_argument("port", type=int)
+    args = parser.parse_args()
+    host = args.host
+    port = args.port
+    return host, port
+
+def signal_handler(sig, frame):
```

Comment:
Type hinting in Python3 should be used - https://google.github.io/styleguide/pyguide.html#221-type-annotated-code

Reply:
I made very little updates to the python code but I will definitely not code this way for the exam